src/AutoRotatingCarousel.js

```diff
@@ -185,15 +185,15 @@ export default class AutoRotatingCarousel extends Component {
             </Carousel>
           </Paper>
           <div style={landscape ? {minWidth: 300, maxWidth: 'calc(50% - 48px)', padding: 24, float: 'right'} : null}>
-            <div style={landscape ? {...style.footerLandscape, ...this.props.footerLandscapeStyle} : {...style.footer, ...this.props.footerStyle}}>
+            <div style={landscape ? style.footerLandscape : style.footer}>
```

Comment:
You forgot to merge the `footerStyle` here; it should be `landscape ? { ...style.footerLandscape, ...footerStyle } : { ...style.footer, ...footerStyle }`